### ISSUE TITLE
Investigate how frontend can efficiently display multiple pool versions

### ISSUE THREAD

**Issue by jrhea** [delvtech/hyperdrive-frontend#1190]

**Comment by ryangoree** [comment]
The plan is to integrate new versions seamlessly where possible. If a new version has an ABI change that is incompatible with previous versions, we'll rev all packages as normal, then the UI will install the both packages (using a package alias for the older version). Custom version logic will be consolidated in hooks and custom components will be created ONLY when necessary. In these cases, a version picker will be added close to the component (e.g., the list of pools) which can be used to switch.

![Image](https://github.com/delvtech/hyperdrive-frontend/assets/3289505/3bb311c1-9122-4379-b8c9-d5a858cf4e2f)

TL;DR: Merge all versions together until we can't.
